build.gradle

```diff
@@ -23,6 +23,12 @@ allprojects {
   apply plugin: 'java' // do not add kotlin plugin here, it will add unnecessary Kotlin runtime dependencies
   apply plugin: 'jacoco'
 
+  configurations.all {
+    resolutionStrategy {
+      force "org.web3j:core:${libs.versions.web3j.get()}"
```

Comment:
Yes, it works. The reason I added this is because we have multiple versions of web3j, through transitive dependencies and when coding/debugging intellIJ was linking to the wrong one. So did this a quick workaround and ended up pushing by accident.

I am aware that this is not a solution that scales! We shall use proper management like spring plugin but I don't have time to address it now.
I am happy to revert or keep it since it does not cause any harm anyway.

Reply:
If it doesn't cause issues for you, I'd like to remove it, because it creates a bad precedent